server/models/preference.go

```diff
@@ -68,6 +68,7 @@ type Preference struct {
 	AnonymousPerfResults      bool                   `json:"anonymousPerfResults"`
 	UpdatedAt                 time.Time              `json:"updated_at,omitempty"`
 	UsersExtensionPreferences map[string]interface{} `json:"usersExtensionPreferences,omitempty"`
+	GetStarted                []int                  `json:"getStarted,omitempty"`
```

Comment:
@senali-d Instead of storing this like this way, can you please make changes in meshery-cloud, to store it as 
`meshery-cloud-prefs: {
getStarted: ...
}` 
so that in future other prefs can also be handled.

When you do so signal me I also need to update here.

Reply:
@MUzairS15 Raised a pr. saved prefs under `mesheryCloudPrefs`

Reply:
Thanks, I will add changes here.